docs/src/main/sphinx/installation/kubernetes.rst

```diff
@@ -0,0 +1,342 @@
+   ``values.yaml`` file to update default behavior. Run the following command to
+   use all default values and create a cluster called ``my-trino-cluster``.
+
+   .. code-block:: text
+
+       helm install my-trino-cluster trino/trino
+
+   To override the default values, you can use the ``values.yaml`` created in
+   the previous step.
+
+   .. code-block:: text
+
+       helm install -f values.yaml my-trino-cluster trino/trino
+
+   You should see output as follows:
+
+   .. code-block:: text
+
+       NAME: my-trino-cluster
+       LAST DEPLOYED: Tue Sep 13 14:12:09 2022
+       NAMESPACE: default
+       STATUS: deployed
+       REVISION: 1
+       TEST SUITE: None
+       NOTES:
+       Get the application URL by running these commands:
+         export POD_NAME=$(kubectl get pods --namespace default -l "app=trino,release=my-trino-cluster,component=coordinator" -o jsonpath="{.items[0].metadata.name}")
+         echo "Visit http://127.0.0.1:8080 to use your application"
+         kubectl port-forward $POD_NAME 8080:8080
+
+   This output will differ based on your ``values.yaml`` configuration and
+   cluster name. For example, the port ``8080`` is set by the
+   ``.Values.service.port`` in the ``values.yaml``.
+
+#. Run the following command to check that all pods, deployments, and services are
+   running properly.
+
+   .. code-block:: text
+
+      kubectl get all
+
+   You should see expect to see output that shows running pods
+   and deployments and replicasets in the ready state.
+
+   .. code-block:: text
+
+       NAME                                               READY   STATUS    RESTARTS   AGE
+       pod/my-trino-cluster-coordinator-bfb74c98d-rnrxd   1/1     Running   0          161m
+       pod/my-trino-cluster-worker-76f6bf54d6-hvl8n       1/1     Running   0          161m
+       pod/my-trino-cluster-worker-76f6bf54d6-tcqgb       1/1     Running   0          161m
+
+       NAME                       TYPE        CLUSTER-IP    EXTERNAL-IP   PORT(S)    AGE
+       service/my-trino-cluster   ClusterIP   10.96.25.35   <none>        8080/TCP   161m
+
+       NAME                                           READY   UP-TO-DATE   AVAILABLE   AGE
+       deployment.apps/my-trino-cluster-coordinator   1/1     1            1           161m
+       deployment.apps/my-trino-cluster-worker        2/2     2            2           161m
+
+       NAME                                                     DESIRED   CURRENT   READY   AGE
+       replicaset.apps/my-trino-cluster-coordinator-bfb74c98d   1         1         1       161m
+       replicaset.apps/my-trino-cluster-worker-76f6bf54d6       2         2         2       161m
+
+   * The top three lines are pods. These are the actual Trino containers running.
+
+   * The next line is the service. This is the method Kubernetes makes internal
+     services like the one running on the coordinator accessible outside of the
```

Comment:
No, a `ClusterIP` service does not make the service accessible outside. It only makes it accessible inside the cluster via a name, instead of IPs.

Reply:
I'd love to pick your brain about these...I've been confused on services for a while now.